pkg/client/fake/client.go

```
@@ -380,6 +401,31 @@ func (t versionedTracker) Update(gvr schema.GroupVersionResource, obj runtime.Ob
 	return t.ObjectTracker.Update(gvr, obj, ns)
 }
 
+func isCustomResource(gvk schema.GroupVersionKind) bool {
+	return !scheme.Scheme.Recognizes(gvk)
+}
+
+func didNonMetadataFieldChange(old runtime.Object, new runtime.Object) bool {
+	oldUnstructured, _ := runtime.DefaultUnstructuredConverter.ToUnstructured(old)
+	newUnstructured, _ := runtime.DefaultUnstructuredConverter.ToUnstructured(new)
```

Why are we ignoring errors here?

---

My bad. Added error handling.